src/index.tsx

```diff
@@ -20,10 +20,6 @@ if (process.env.NODE_ENV === 'development') {
       root = render(<App />, document.body, root);
     });
   });
-} else if ('serviceWorker' in navigator && location.protocol === 'https:') {
-  addEventListener('load', () => {
-    navigator.serviceWorker.register(__webpack_public_path__ + 'sw.js');
-  });
```

Comment:
I don't think it's time to add a service worker yet. Also we wouldn't want it to be https-only, as testing locally is important.